contrib/auth/acl/service.go

```diff
@@ -895,8 +895,8 @@ func (s *AuthService) Authorize(ctx context.Context, req *auth.AuthorizationRequ
 	if err != nil {
 		return nil, err
 	}
-
-	allowed := auth.CheckPermissions(ctx, req.RequiredPermissions, req.Username, policies)
+	perm := &auth.MissingPermissions{}
```

Comment:
Going back to prev question - what are you doing with `perm` after `auth.CheckPermissions` ? I don't see it being used.